### PR TITLE
DTSPO-5340 Add Environment docs

### PR DESCRIPTION
**Pull request by zc-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/DTSPO-5340

### Change description ###

**Does this PR introduce a breaking change?**

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by zc-hmcts** [comment]
Screenshot 2021-11-01 at 16 26 19

File: source/ways-of-working/guides/environments.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# <%= current_page.data.title %>
+
+
+
+| Shared Services Environment | CNP Environment | Use case &nbsp; | Path to live | Additional Information &nbsp; &nbsp; |
+| - | - | - | - | - | - |
+| Prod            | Prod       | Live services | Yes |  |
+| staging         | AAT        | Automated Acceptance Testing before moving to Prod | Yes |  |
+| Demo            | Demo       | External demonstrations (no VPN required) | No | Used for demonstrating to external stakeholders |
```
Comment (timja) [inline source/ways-of-working/guides/environments.html.md.erb:16]: could you make 'Used for demonstrating to external stakeholders' a list and move no vpn required to the additional information as well

File: source/ways-of-working/guides/environments.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+last_reviewed_on: 2021-11-01
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+
+
+| Shared Services Environment | CNP Environment | Use case &nbsp; | Path to live | Additional Information &nbsp; &nbsp; |
```
Comment (timja) [inline source/ways-of-working/guides/environments.html.md.erb:12]: do you need these non breaking spaces?

```suggestion
| Shared Services Environment | CNP Environment | Use case | Path to live | Additional Information |
```

**Review by timja — COMMENTED** [review]
I think this belongs in Concepts and standards not in guides

File: source/ways-of-working/guides/environments.html.md.erb (hunk 2 above)
Comment (njrich28) [reply source/ways-of-working/guides/environments.html.md.erb:12]:
```suggestion
| Shared Services Environment | CFT Environment | Use case &nbsp; | Path to live | Additional Information &nbsp; &nbsp; |
```

File: source/ways-of-working/guides/environments.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+| - | - | - | - | - | - |
+| Prod            | Prod       | Live services | Yes |  |
+| staging         | AAT        | Automated Acceptance Testing before moving to Prod | Yes |  |
+| Demo            | Demo       | External demonstrations (no VPN required) | No | Used for demonstrating to external stakeholders |
+| ITHC            | ITHC       | IT Health Check, used for penetration testing | No |  |
+| Test            | Perftest   | Performance testing | No | |
+| Dev             | Preview    | Development of software changes | Yes | <ul> <li>Destroyed nightly</li> <li>Pull requests are deployed</li>​ <br> <li>Automatically tested with functional tests​</li> <br> <li>Application points to AAT instance for dependant services</li> </ul> |
```
Comment (njrich28) [inline source/ways-of-working/guides/environments.html.md.erb:19]: Maybe not say 'development' here - we're trying to move teams away from using it as a personal dev environment.

```suggestion
| Dev             | Preview    | Pre-merge automated testing and Pull Request checks | Yes | <ul> <li>Destroyed nightly</li> <li>Pull requests are deployed</li>​ <br> <li>Automatically tested with functional tests​</li> <br> <li>Application points to AAT instance for dependant services</li> </ul> |
```

**Comment by zc-hmcts** [comment]
Screenshot 2021-11-02 at 09 20 39

File: source/ways-of-working/guides/index.html.md.erb [hunk 4]
```diff
@@ -7,4 +7,5 @@ weight: 98
 
 # <%= current_page.data.title %>
 
-- [Accessing apps via the VPN](connect-via-vpn.html)
\ No newline at end of file
+- [Accessing apps via the VPN](connect-via-vpn.html)
+- [Environments](environments.html)
```
Comment (timja) [inline source/ways-of-working/guides/index.html.md.erb:11]: can we move this to `Concepts and standards`

**Review by timja — APPROVED** [review]

**Review by msl8r — APPROVED** [review]

File: source/ways-of-working/guides/index.html.md.erb (hunk 4 above)
Comment (zc-hmcts) [reply source/ways-of-working/guides/index.html.md.erb]: moved

**Pull request merged**